app/views/admin/team_members/_form.html.haml

```diff
@@ -8,12 +8,19 @@
     %fieldset
       %legend Basics
       = f.input :name, required: true
+      = f.input :title
+      Leader
```

Comment:
What is this line doing? If it the label for the `is_leader` input, that should be specified as the label on the input.